index.js

```diff
@@ -12,16 +12,26 @@ module.exports = function(content) {
 	this.cacheable && this.cacheable();
 
 	var query = loaderUtils.getOptions(this);
+	var configKey = query.config || "svgAsSymbolLoader";
```

Comment:
Shouldn't `svgAsSymbolLoader` be documented in the readme? Or it is something which is very common and does not need documentation?